requirements-linux-build.txt

```diff
@@ -0,0 +1,2 @@
+# The Ubuntu GUI exits immediately without error in 6.11.0.
+cx_Freeze==6.10
```

Comment:
This is for all of the supplemental things that can't be matched up between operating systems. This isn't really a dev dependency, I'm kind of wondering if we want to just hardcode in the various workflows instead. I'll leave up to you though

Reply:
That makes sense. All of these build requirements files should work the same though. Can you remove the `-r requirements-dev.txt` from `requirements-windows-build.txt` and update the pip calls to it? Also would be good to add `cx_Freeze` to the dependabot ignore list since we're using different versions.

Reply:
Yep that makes a lot of sense, will do!